testbed/go.mod

```diff
@@ -6,9 +6,10 @@ require (
 	contrib.go.opencensus.io/exporter/jaeger v0.1.1-0.20190430175949-e8b55949d948
 	github.com/open-telemetry/opentelemetry-service v0.0.0-20190625135304-4bd705a25a35
 	github.com/shirou/gopsutil v2.18.12+incompatible
-	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.3.0
-	go.opencensus.io v0.22.0
+	github.com/shirou/w32 v0.0.0-20160930032740-bb4de0191aa4 // indirect
```

Comment:
why adding this one? it is used by github.com/shirou/gopsutil/process that is used on testbed

```terminal
$ go mod why github.com/shirou/w32
# github.com/shirou/w32
github.com/open-telemetry/opentelemetry-service/testbed/testbed
github.com/shirou/gopsutil/process
github.com/shirou/w32
```